proto/store/user_setting.proto

```diff
@@ -15,6 +23,7 @@ message UserSetting {
     string appearance = 5;
     string memo_visibility = 6;
     string telegram_user_id = 7;
+    string memo_mode = 8;
```

Comment:
How about `bool compact_view = 8`?

Reply:
Yes it's a great suggestion. I will work on it and push the changes soon. Thank you.